lib/gooddata/lcm/types/class/class.rb

```diff
@@ -0,0 +1,21 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2016 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require_relative '../base_type'
+
+module GoodData
+  module LCM2
+    module Type
+      class ClassType < BaseType
+        CATEGORY = :class
+
+        def to_s
+          self.class.name.split('::').last
```

Comment:
could we use this one? https://github.com/gooddata/gooddata-ruby/pull/745/files#diff-189a95a928410eb41446e31c3bdea6cdR2